.github/workflows/dotnet-tests.yml

```diff
@@ -0,0 +1,67 @@
+name: Dotnet Framework Tests
+on: [push, pull_request]
+
+jobs:
+  run-babelfish-dotnet-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        id: checkout
+
+      - name: Install Dependencies
+        id: install-dependencies
+        if: always()
+        uses: ./.github/composite-actions/install-dependencies
+
+      - name: Build Modified Postgres
+        id: build-modified-postgres
+        if: always() && steps.install-dependencies.outcome == 'success'
+        uses: ./.github/composite-actions/build-modified-postgres
+      
+      - name: Compile ANTLR
+        id: compile-antlr
+        if: always() && steps.build-modified-postgres.outcome == 'success'
+        uses: ./.github/composite-actions/compile-antlr
+      
+      - name: Build Extensions
+        id: build-extensions
+        if: always() && steps.compile-antlr.outcome == 'success'
+        uses: ./.github/composite-actions/build-extensions
+      
+      - name: Install Extensions
+        id: install-extensions
+        if: always() && steps.build-extensions.outcome == 'success'
+        uses: ./.github/composite-actions/install-extensions
+        with:
+          TEST_USER: 'dotnet_user'
+          TEST_DB: 'dotnet_testdb'
```

Comment:
All of these steps are duplicated; can you break them out as their own file? (I don't think we need different database or user for each different test)

Reply:
This was intentional, this way all of the steps show up in their individual tabs under the Actions which makes it easier to debug troubleshoot. When i isolate these common steps to a single composite action file they show up as a single step.

Reply:
Bleg; it'd be really nice to find a way to avoid that. I'm guessing there's some way to do it; we'd just have to find it. In any case, that can wait.